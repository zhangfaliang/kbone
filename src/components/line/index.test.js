import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import Line from './index';

describe('Line', async () => {
  it('base', () => {
    expect.assertions(1);
    const wrapper = render(<Line />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('prefixClsNull', () => {
    expect.assertions(1);
    const wrapper = render(<Line prefixCls={null} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
