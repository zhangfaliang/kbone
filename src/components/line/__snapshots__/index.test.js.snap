// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line base 1`] = `
<div
  class="row-line"
/>
`;

exports[`Line prefixClsNull 1`] = `
<div
  class="row-line"
/>
`;
